hikari/impl/entity_factory.py

```diff
@@ -2226,6 +2276,13 @@ def deserialize_select_menu(self, payload: data_binding.JSONObject) -> message_m
             is_disabled=payload.get("disabled", False),
         )
 
+    def deserialize_text_input(self, payload: data_binding.JSONObject) -> modal_interactions.ModalInteractionTextInput:
```

Comment:
```suggestion
    def deserialize_modal_text_input(self, payload: data_binding.JSONObject) -> modal_interactions.ModalInteractionTextInput:
```

Reply:
Unfortunately, I have decided to decline this suggestion.